src/components/Splash.js

```diff
@@ -229,34 +269,20 @@ export default ({
             </ul>
           </div>
 
-          <iframe src="https://guild.host/embeds/guild/torontojs/card"
-            loading="lazy"
-            style={{ width: `450px`, height: `360px`, border: `none`, overflow: `hidden` }}
-          />
+          <div id="guild-card-root" style={GuildCardRootStyle} />
```

Comment:
Not sure if it'll work like this:

```suggestion
          <GuildCardRoot id="guild-card-root" />
```

Or if it needs to be like this:

```suggestion
          <GuildCardRoot>
            <div id="guild-card-root" />
          </GuildCardRoot>
```

Reply:
Fixed in 1359ff2